commcare_translations.py

```diff
@@ -54,5 +54,7 @@ def dumps(dct):
             val = u'\u00A0'
         # get rid of newlines
         val = val.replace('\n', '\\n')
+        # escape starting # character
+        val = re.sub('^#', '\#', val)
```

Comment:
@dannyroberts does this make sense to you? i'm not totally familiar with this part of the code